recipes/vcflib/build.sh

```
@@ -4,15 +4,20 @@ set -ex
 OS=$(uname)
 ARCH=$(uname -m)
 
+cp -rf "${RECIPE_DIR}/vcflib.pc.in" "${SRC_DIR}"
+
 if [[ "${OS}" == "Darwin" && "${ARCH}" == "x86_64" ]]; then
 	echo $(pwd)/zig-macos-x86_64-*
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)